### PR TITLE
feat(docs): add support for multilingual documentation (HINDI) #fixes…

### LINKED ISSUE #35: [Traslations] Translate README.md to हिन्दी (hi) locale
**Is your feature request related to a problem? Please describe.**
Currently the documentation supports English only

**Describe the solution you'd like**
For example:
- Add `docs` dir at the root of project. And put the `{locale}/README.md` into `docs` dir

**Additional context**
See the contributing [guide](https://github.com/dipiash/eslint-plugin-nimbus-clean/blob/main/CONTRIBUTING.md).

### PR DESCRIPTION
**Pull request by aditya-armal**
Close #35

### CODE REVIEW

File: docs/hi/Readme.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+![GitHub CI](https://github.com/dipiash/eslint-plugin-nimbus-clean/actions/workflows/ci.yml/badge.svg)
+[![npm version](https://badge.fury.io/js/eslint-plugin-nimbus-clean.svg?v=0.2.4)](https://badge.fury.io/js/eslint-plugin-nimbus-clean)
+[![HitCount](https://hits.dwyl.com/dipiash/eslint-plugin-nimbus-clean.svg?style=flat-square)](http://hits.dwyl.com/dipiash/eslint-plugin-nimbus-clean)
+
+`README.md`: [EN](/README.md) | [日本語](/docs/ja/README.md) | [繁體中文](/docs/zh-tw/README.md)
```
Comment (dipiash) [inline docs/hi/Readme.md:5]: Could you add the link on this readme in all others docs/ and root readmes please?

**Review by dipiash — CHANGES_REQUESTED** [review]

**Comment by dipiash** [comment]
Could you add the new command in script file for auto TOC generation, please?

https://github.com/dipiash/eslint-plugin-nimbus-clean/blob/main/scripts/prepareTOCinReadme.sh

File: docs/hi/Readme.md (hunk 1 above)
Comment (Aditya-Armal) [reply docs/hi/Readme.md:5]: Sure Dipiash!

**Comment by Aditya-Armal** [comment]
Done @dipiash

**Review by dipiash — APPROVED** [review]

**Comment by dipiash** [comment]
Thank you @aditya-armal!

**Pull request merged**